index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Flex box</title>
+    <link rel="stylesheet" href="style.css">    
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <div class="toolbox">
+
+        <h1>My Logo</h1>
+        <img src="./images/hamburguer.png" alt="hamburguer">
+    </div>
+    <div class="principal">
+        <h2>I’m Riziki Glad to see you!</h2>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h3>LET’S CONNECT</h3>
+        <ul class="social">
+            <li><img src="./images/Linkedin.svg" alt="linked"></li>
+            <li> <img src="./images/twitter.svg" alt="twitter"></li>
+            <li><img src="./images/m.svg" alt="m"></li>
+            <li><img src="./images/hand.svg" alt="hand"></li>
+            <li><img src="./images/github.svg" alt="github"></li>
```

Comment:
- [ ] Each of your icons has a link (`a` tag) which means you  should wrap your image tag into an `a` tag